tests/builtins/test_iter.py

```diff
@@ -12,9 +12,9 @@ class BuiltinIterFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
         'test_bytearray',
         'test_bytes',
         'test_class',
-        'test_dict',
         'test_frozenset',
         'test_list',
+        'test_dict',
```

Comment:
This line seems to have been moved down with no real effect – could you undo this change?